packages/block-editor/src/components/block-list/classic.scss

```diff
@@ -1,3 +1,7 @@
+.editor-styles-wrapper {
+	background: $white;
+}
+
 // This needs specificity to override the editor styles.
 .editor-styles-wrapper .wp-block {
 	margin-left: auto;
```

Comment:
@youknowriad @jasmussen Where should these "classic" alignment styles and default margins etc. best be located? This needs to move out of the `edit-post` package to either `block-library` or `block-editor` because we don't want to load the `edit-post` styles in the iframe.